src/wp-includes/class-wp-block-styles-registry.php

```diff
@@ -36,10 +36,20 @@ final class WP_Block_Styles_Registry {
 	 *
 	 * @since 5.3.0
 	 *
+	 * @link https://developer.wordpress.org/block-editor/reference-guides/block-api/block-styles/
+	 *
 	 * @param string $block_name       Block type name including namespace.
-	 * @param array  $style_properties Array containing the properties of the style name, label,
-	 *                                 is_default, style_handle (name of the stylesheet to be enqueued),
-	 *                                 inline_style (string containing the CSS to be added).
+	 * @param array  $style_properties {
+	 *     Array containing the properties of the style.
+	 *
+	 *     @type string $name         The identifier of the style used to compute a CSS class.
+	 *     @type string $label        A human-readable label for the style.
+	 *     @type string $inline_style Inline CSS code that registers the CSS class required
+	 *                                for the style.
+	 *     @type string $style_handle The handle to an already registered style that should be
+	 *                                enqueued in places where block styles are needed.
```

Comment:
Those two parameters are optional so we could leave a note about it. Usually, one of them would be used to pass CSS styles. However, you could also skip them and provide CSS styles in any stylesheet or with an inline tag.